zulip_bots/zulip_bots/run.py

```diff
@@ -114,7 +114,15 @@ def main() -> None:
     if args.provision:
         provision_bot(os.path.dirname(bot_path), args.force)
 
-    lib_module = import_module_from_source(bot_path, bot_name)
+    try:
+        lib_module = import_module_from_source(bot_path, bot_name)
+    except ImportError as e:
+        dep_err_msg = ("ERROR: It seems that the dependencies for {bot_name} are not installed. "
+                       "Please include the --provision option when running zulip-run-bot, like so:\n"
+                       "    zulip-run-bot {bot_name} --provision --config-file=<PATH_TO_CONFIG>")
```

Comment:
We should say "If you'd like us to install these dependencies, run ..."

And probably also just print out the list of them at the end of the error message?